demo/eip/routes/authenticate-to-openmrs-route.xml

```diff
@@ -25,22 +25,5 @@
             <simple>${exchangeProperty.basic-openmrs-authentication}</simple>
         </setHeader>
 
-        <!-- If SSO is enabled then override basic authentication in favor of Oauth -->
-        <setBody>
-            <simple></simple>
-        </setBody>
-        <process ref="oauthProcessor" />
-        <choice>
-            <when>
-                <simple>${body} != null &amp;&amp; ${body} != ""</simple>
-                <setHeader name="Authorization">
-                    <simple>${body}</simple>
-                </setHeader>
-            </when>
-        </choice>
-
-        <setBody>
-            <simple>${exchangeProperty.body}</simple>
-        </setBody>
```

Comment:
Is authentication being handled by the eip (pro) client itself?

Reply:
That's the idea, the OAuth2 stuff will be handled by `eip-pro-client`.